kinder/pkg/cluster/manager/actions/kubeadm-upgrade.go

```diff
@@ -131,7 +128,9 @@ func kubeadmUpgradeApply(c *status.Cluster, cp1 *status.Node, upgradeVersion *K8
 		"upgrade", "apply", "-f", fmt.Sprintf("v%s", upgradeVersion), fmt.Sprintf("--v=%d", vLevel),
 	}
 	if patchesDir != "" {
-		applyArgs = append(applyArgs, "--experimental-patches", constants.PatchesDir)
+		if cp1.MustKubeadmVersion().LessThan(constants.V1_22) {
+			applyArgs = append(applyArgs, "--patches", constants.PatchesDir)
```

Comment:
```suggestion
			applyArgs = append(applyArgs, "--experimental-patches", constants.PatchesDir)
```

should be the old flag for < 1.22